spec/ioki/http_adapter_spec.rb

```diff
@@ -68,6 +70,14 @@
 
       it { is_expected.not_to include 'X-Client-Identifier' }
     end
+
+    context 'when bleeding edge is not set' do
+      before do
+        config.api_bleeding_edge = false
+      end
+
+      it { is_expected.not_to include 'X-Bleeding-Edge' => config.api_bleeding_edge }
+    end
```

Comment:
What's the reasoning for testing the default with a special test, while testing the special case as default? I'd expect it to be the other way around (granted, this is a bit of a nitpick)

Reply:
I was thinking too much about my own use case :grinning: I agree, will switch it!